src/main/java/nitezh/ministock/activities/widget/WidgetView.java

```diff
@@ -64,6 +68,7 @@ of this software and associated documentation files (the "Software"), to deal
     private final String quotesTimeStamp;
     private final Context context;
     private final HashMap<ViewType, Boolean> enabledViews;
+    public MyData theFkingList = new MyData();
```

Comment:
Maybe change the name to something more kosher?

Reply:
I agree. I didn't call it that, lol